examples/tls_server-rs/ta/test-ca/ecdsa/ca.cert

```diff
@@ -1,12 +1,12 @@
 -----BEGIN CERTIFICATE-----
-MIIByjCCAVCgAwIBAgIUSA11/39PY7uM9Nc2ITnV1eHzaKYwCgYIKoZIzj0EAwIw
-HDEaMBgGA1UEAwwRcG9ueXRvd24gRUNEU0EgQ0EwHhcNMTkwNjA5MTcxNTEyWhcN
-MjkwNjA2MTcxNTEyWjAcMRowGAYDVQQDDBFwb255dG93biBFQ0RTQSBDQTB2MBAG
-ByqGSM49AgEGBSuBBAAiA2IABLsXWEKs2xXCgW1OcC63pCPjQo0q3VnPc1J24n6m
-Xwxpg398nzR4n3iHcYA0pKgEneBstSOsXOhbNZ09DAvEr3iSc8ByWWntEbWVjY3g
-9Kt6Q6Y1sXGkaUIiP9be5lIQRaNTMFEwHQYDVR0OBBYEFKD72TTU/GXhb3/D1/Z7
-hD/ZG6lKMB8GA1UdIwQYMBaAFKD72TTU/GXhb3/D1/Z7hD/ZG6lKMA8GA1UdEwEB
-/wQFMAMBAf8wCgYIKoZIzj0EAwIDaAAwZQIxAL9FtbNV7i9trxukhakfTvbXCHgE
-2pIOT5r/Vc5kSrPU4vJu2MOJz6X/JCX15IbZlQIwJxYfsD8QTQf8J9bP9Pq4SY71
-obja/vQ6UBixlRB5vDSG0UuukL4kzlyUKpHkwUcj
+MIIBvDCCAUKgAwIBAgIUcfobdnUAGg7+ZKWcoJDhubf76iEwCgYIKoZIzj0EAwIw
+FTETMBEGA1UEAwwKdGVzdHNlcnZlcjAeFw0yNTA4MTMwMzQ5MTFaFw0zNTA4MTEw
+MzQ5MTFaMBUxEzARBgNVBAMMCnRlc3RzZXJ2ZXIwdjAQBgcqhkjOPQIBBgUrgQQA
+IgNiAARuhizLUdjzt2IufRL1ELnTris+0kHEkc1WYS/vSaj+8dMRZ9/MW8ScsMjn
+zs0K6XMme4o3TuPgFm3MdiWlIwHBN4CjkPN3u0U3N3Lw1DHvl/oUMfQ/mgBt9/Cz
+osOnSx2jUzBRMB0GA1UdDgQWBBRIkFT2dS2wya07AoKzpoL0E6SO/zAfBgNVHSME
+GDAWgBRIkFT2dS2wya07AoKzpoL0E6SO/zAPBgNVHRMBAf8EBTADAQH/MAoGCCqG
+SM49BAMCA2gAMGUCMQCD4bitceb4NJ9Snf3X3Watg3YlWv5mmeT3K6khxEiypQOd
+/XJC+UvUisbD9srbWLUCMBydSCIzPsS4CGIWQ+lZY1+tqWQ6FVW9iTQ7pYgZSibD
```

Comment:
Is there any specific reason for changing the existing test certs/keys?
It’s good to add a script that explains how to generate these test-CA artifacts. However, for the existing artifacts, we should keep them unchanged and reuse them as a demonstration that the updated TLS example remains compatible.